src/assets/css/myStyles.scss

```diff
@@ -0,0 +1,22 @@
+@charset "utf-8";
```

Comment:
myStyles is not an appropriate name 😄 

use other name like `template` or `custom` or something like this

Reply:
hmm ok!

Reply:
you didn't solve this issue